### DISCUSSION TITLE
NextJS integration

### DISCUSSION THREAD

**Discussion by Roz1c** [soketi/pws#843] [Q&A]
Can soketi be used with NextJS 13? If so is there any example or tutorial available?

**Comment by Roz1c** [comment]
I got it to work. It is very simple.

**Reply by nickchomey** [reply]
Care to share details for others to learn from?

**Reply by Roz1c** [reply]
Sure, so just spin up the soketi docker image and integrate pusher like in this repo https://github.com/joschan21/nextjs-realtime-chat/search?q=pusher

**Reply by DezyNation** [reply]
Why did you use cluser: "eu"?
And did you use your server's ip address as pusher base url or soketi's subdomain on your server??
Which port number did you use?

**Reply by DezyNation** [reply]
As the documentation suggests, you are not using any wsHost, wssPort or enabledTransports: ['wss']
I found this in your code


src/lib/pusher.ts
<br/>
`
export const pusherClient = new PusherClient(
  process.env.NEXT_PUBLIC_PUSHER_APP_KEY!,
  {
    cluster: 'eu',
  }
)
`
